libc/spec/linux.td

```diff
@@ -287,6 +287,23 @@ def Linux : StandardSpec<"Linux"> {
       ]
   >;
 
+
+  HeaderSpec UniStd = HeaderSpec<
+    "unistd.h",
+    [], // Macros
+    [],
+    [], // Enumerations
+    [
+        FunctionSpec<
+          "pipe",
```

Comment:
ah, good catch, yeah this should be `pipe2`. Sorry I missed it!

Reply:
No worries! This is fixed in [b1e2ea6](https://github.com/llvm/llvm-project/commit/b1e2ea64182ddab6775ab6cfe5168c98c92c1a57).